module/zcommon/zfs_prop.c

```diff
@@ -587,6 +684,10 @@ zfs_prop_init(void)
 	zprop_register_hidden(ZFS_PROP_REMAPTXG, "remaptxg", PROP_TYPE_NUMBER,
 	    PROP_READONLY, ZFS_TYPE_DATASET, "REMAPTXG");
 
+	zprop_register_impl(ZFS_PROP_COMPRESS_LEVEL, "compress_level",
+	    PROP_TYPE_NUMBER, ZIO_COMPLEVEL_INHERIT, NULL, PROP_INHERIT,
+	    ZFS_TYPE_DATASET, "<level>", "COMPLEVEL", B_TRUE,
+	    B_FALSE, NULL);
```

Comment:
@c0d3z3r0 I think @BrainSlayer never executed the advice from an earlier review from Behlendorf here:
https://github.com/zfsonlinux/zfs/pull/8044#discussion_r226809133